response_writer.go

```diff
@@ -13,7 +10,7 @@ import (
 // if the functionality calls for it.
 type ResponseWriter interface {
 	http.ResponseWriter
-	http.Flusher
```

Comment:
never a good idea to remove a method from a public interface

Reply:
Based on my experience, this interface implementing would never be used directly as `w.Flush()` but likes below
```go
if f, ok := w.(http.Flusher); ok {
     f.Flush()
}
```
This change does not affect the contract, and follows the issue #265

Reply:
I agree that calling code typically looks like that and so the impact is likely to be minimal, but I still think this interface change is a breaking change and so we should put it in version 3 when releasing.